.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: python
 python:
-  - "2.6"
   - "2.7"
   - "3.4"
+  - "3.5"
```

Comment:
It wasn't released when I was fixing up @pudo's branch last year.

If the tests also pass for it, then, of course, it could be added.